### ISSUE TITLE
Layer names show `]]` (from the link)

### ISSUE THREAD

**Issue by Moonbase59** [valentine195/obsidian-leaflet-plugin#196]
![Auswahl_172](https://user-images.githubusercontent.com/3706922/128442412-8309e12f-7061-4cf4-83ec-37e0d3520c83.png)

**Comment by valentine195** [comment]
Can you show me the source code? This is not reproducing on my end.

**Comment by Moonbase59** [comment]
Sure. The top one. Test file from the time we introduced named layers:

````
---
tags: [map, game, fantasy]
---

# Khorvaire map

```leaflet
id: khorvaire
image:
    - [[Khorvaire.jpg|Khorvaire]]
    - [[Khorvaire-Talenta Plains2.png|Talenta Plains]]
height: 500px
lat: 50
long: 50
zoomDelta: 0.5
defaultZoom: 6.5
unit: miles
scale: 400
marker: Check, -3.8681, 6.6143
#marker: Current, 50, 50
```

```leaflet
id: khorvaire2
image: [[Khorvaire-background.png]]
imageOverlay:
  - [ [[Khorvaire-Talenta Plains.png]] ]
height: 500px
lat: 50
long: 50
zoomDelta: 0.5
defaultZoom: 6.5
unit: miles
scale: 400
marker: Check, -3.8681, 6.6143
#marker: Current, 50, 50
```
````

**Comment by valentine195** [comment]
Ah, it’s from the alias, forgot about that!

**Comment by Moonbase59** [comment]
Still in 3.24.3

**Comment by Moonbase59** [comment]
Confirm fixed in 3.24.4 :+1:
